src/lib/theme/plugin.ts

```diff
@@ -13,6 +13,7 @@ export const variables = {
   '--color-interactive-hover': rgb(getColor('indigo', 700)),
 
   '--color-text-black': rgb(colors.black),
+  '--color-text-white': rgb(colors.white),
```

Comment:
I think we can just use off-white instead, which already exists as `--color-text-inverse`